packages/reports/config/environment.js

```diff
@@ -15,6 +15,7 @@ module.exports = function(/* environment, appConfig */) {
         timeout: 90000
       },
       predefinedIntervalRanges: {
+        hour: ['P1D', 'P7D', 'P14D', 'P30D'],
```

Comment:
Do we want to include a list of sub day presets?

Reply:
This might kick them over to the advanced section, so it might feel strange in the same list